packages/core/src/utils/node.ts

```diff
@@ -0,0 +1,22 @@
+import { DomElement } from 'htmlparser2';
+
+/*
+ * A TextElement is a simple node that does not need complex processing.
+ */
+export type TextElement = DomElement;
+
+/*
+ * MbNode (MarkbindNode) is an element that can be operated on by cheerio and our own node processing
```

Comment:
```suggestion
 * MbNode (MarkBindNode) is an element that can be operated on by cheerio and our own node processing
```

Reply:
Typo here, I suppose MbNode is still ok, or do you think we should change it to MBNode?

Reply:
Hmm I'd prefer MbNode (MBNode feels like too many capital letters). Open to suggestions!